File: Unidad 2-Estructuras de Control/Ejercicio47.c
#include <stdio.h>

#include <conio.h>

main()

{

float sueldo, afp, isss, renta, sn;

char nombre[50];

clrscr();

printf("Introduzca el Nombre del empleado:\n");

scanf("%s", nombre);

printf("Su sueldo es:\n");

scanf("%f", &sueldo);

afp=sueldo*0.07;

isss=sueldo*0.05;

renta=sueldo*0.10;

sn=sueldo-(afp+isss+renta);

printf("El empleado %s\n", nombre);

printf("Posee un sueldo neto de %.2f\n", sn);

getch();

return 0;

}

File: Unidad 2-Estructuras de Control/Ejercicio47.java


import java.util.Scanner;


public class Empleado {
	
	public static void main(String[] args) {
	
	String Nombre = "Jonny";
	double sueldohora=0;
	int edad =0;
	int horasextras =0;
	double sueldohe=0;
	double impuestos=0;
	int hrstrabajadas=0;
	double sueldo=0;
	double sueldofinal=0;
	

		
	Scanner entrada= new Scanner(System.in);
	Scanner entrada2= new Scanner(System.in);

	
	
	System.out.println("Cual es el nombre del Empleado");
	Nombre= entrada2.nextLine();
	
	
	
	System.out.println("Su Edad");
	edad= entrada.nextInt();
	
	
	
	System.out.println("Introduzca el sueldo");
	sueldohora= entrada.nextDouble();
	
	
	System.out.println("Introduce horas trabajadas a la semana");
	hrstrabajadas= entrada.nextInt();
	
	
	if(hrstrabajadas > 40) {
		
		horasextras= hrstrabajadas-40;
		
	}
	
	
	sueldohe= horasextras*sueldohora*2;
	sueldo= sueldohora*(hrstrabajadas+horasextras);
	
	if(sueldo > 20000) {
		
		impuestos= sueldo*0.15;
	}
	
	else {
	if(sueldo>1000){
		
		impuestos= sueldo*0.10;	
	}
	else {
		impuestos = 0;
		
	}
	}
	
	sueldofinal= sueldo-impuestos;
	System.out.println("Nombre: "+Nombre + "Edad: " + edad + "Sueldo Hora: " + sueldo );
	System.out.println("Horas:  "+ hrstrabajadas + "Impuestos: " + impuestos );
	System.out.println("Sueldo Neto  "+ sueldofinal);
	
	
}
}
